### ISSUE TITLE
Hero's Relic Bug

### ISSUE THREAD

**Issue by Usera2** [ValveSoftware/Dota2-Gameplay#5984]
### Description

Hi Everyone

Lately i have noticed that the relic count is bugged somehow and only count just a couple of relics and not the others. 
I put the screen shot of one of my games as CM.
This bug isnt hero specific. For example the count for "stolen spell" of Rubick wont change at all.

### Screenshots

![image](https://user-images.githubusercontent.com/78626849/208131912-5e187433-c898-45c1-becd-613e54eec435.png)
![image](https://user-images.githubusercontent.com/78626849/208131998-29189952-b826-43e1-b71a-fdf61342eb31.png)

**Comment by SohanDaivikReddy** [comment]
yeah same even the kills and kill assists for me wont change pls fix

**Comment by kaaiuta** [comment]
Same thing for a while, like, "2-3 years a while". My maelstrom damage on Arc Warden just stopped ticking, it used to show somewhere around 150KK damage, but now its stuck at about 90KK

**Comment by Usera2** [comment]
Screenshot of a game with Rubick as another example
![image](https://user-images.githubusercontent.com/78626849/208218038-f70e5545-70c0-4345-a209-7d974686a30a.png)
![image](https://user-images.githubusercontent.com/78626849/208218048-a90e7ea5-f787-41d6-8c33-51b89c333801.png)
